features/input_validation.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Sign up Input Validation
+  As a user that inserts data into platform
+  In order to know how to fill in the right data
+  I want to have fill in support messages
+
+  Scenario: First Name can't be blank
+    Given a new user a with a blank first name field
+    When a tries to sign up
+    Then a should not be able to sign up
+    And the page should show Vorname muss ausgefüllt werden
+
+  Scenario: Last Name can't be blank
+    Given a new user b with a blank last name field
+    When b tries to sign up
+    Then b should not be able to sign up
+    And the page should show Nachname muss ausgefüllt werden
+
+  Scenario Outline: Password should be secure
+    Given a new user b with password <password>
+    When b tries to sign up
+    Then b should not be able to sign up
+    And the page should show <error message>
+
+
```

Comment:
Space — the final frontier. These are the voyages of the starship `develop`. Its ongoing mission — to program strange new apps — to seek out new smells and new optimizations — to boldly code where no man has coded before.

You could just use this example as a scenario specific example